### ISSUE TITLE
Warning: componentWillReceiveProps is deprecated and will be removed in the next major version.

### ISSUE THREAD

**Issue by SnehalAgrawal** [joshjhargreaves/react-native-event-calendar#34]
- Warning: Failed child context type: Invalid child context `virtualizedCell.cellKey` of type `number` supplied to `CellRenderer`, expected `string`.

- Warning: componentWillReceiveProps is deprecated and will be removed in the next major version. Use static getDerivedStateFromProps instead. Please update the following components: DayView

**Comment by Aishwaryashetty** [comment]
yeah, am getting the same error as well. It crashes on the loading of the calendar page.

**Comment by Yeloda** [comment]
Have you found a solution for this problem ? i am getting this error on loading
